scripts/lindtool.sh

```diff
@@ -274,17 +274,15 @@ case $1 in
         unset LD_LIBRARY_PATH
         echo -e "${GREEN}$make_cmd${RESET}"
         if [ "$pmode" -eq 0 ]; then
-            eval "$make_glibc_cmd"
+            bazel build //:make_glibc
         fi
         ;;
     make_all)
          # Makes glibc, compiles wasmtime, compiles rawposix
         unset LD_LIBRARY_PATH
         echo -e "${GREEN}$make_cmd${RESET}"
         if [ "$pmode" -eq 0 ]; then
-            eval "$make_glibc_cmd"
-            eval "$compile_wasmtime_cmd"
-            eval "$compile_rawposix_cmd"
+            bazel build //:make_glibc //:make_rawposix //:make_wasmtime
```

Comment:
Perhaps it would be better here if we added a bazel make_all rule that does all of these things to mimic what was here previously.  Functionally the same, but cleaner to read.